src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -72,12 +72,14 @@ import {
   requireOneSelected,
 } from '../validations';
 import { validateBooleanGroup } from '../../../common/schemaform/validation';
+import PhoneNumberWidget from '../../../common/schemaform/widgets/PhoneNumberWidget';
 
 const {
   treatments: treatmentsSchema,
   privateRecordReleases,
   serviceInformation,
   standardClaim,
+  // veteran
```

Comment:
Is this needed as a placeholder for anything, or is it a vestige of testing?

Reply:
It might be needed, but it's not like it's hard to put back if needed. Depends on how we structure and import the schema. Going to take it out for now.